Blog: announce Dandelion++ (#942)
monero-project/monero-site: _posts/2020-04-18-dandelion-implemented.md

---
layout: post
title: "Another privacy-enhancing technology added to Monero: Dandelion++"
summary: Dandelion++ has been integrated and will be available in the next release
tags: [community]
author: ErCiccione
---

> The Pull Request which adds Dandelion++ [has been merged](https://github.com/monero-project/monero/pull/6314). This new feature changes the way transactions are propagated in the network, making much harder to link an IP address with a transaction.

Reviewer: ```suggestion
The Pull Request which adds Dandelion++ [has been merged](https://github.com/monero-project/monero/pull/6314). This new feature changes the way transactions are propagated to the network, making it much harder to link a transaction to an IP address.
```

> The feature is now part of the code and will be included in the next release of the Monero software, but it's already possible to test it compiling the CLI by yourself (see [instructions on the repository](https://github.com/monero-project/monero#compiling-monero-from-source)). Be aware that to work as intended, a node running Dandelion++ needs to communicate with other nodes which integrate the feature. So, you can run a Dandelion++ node on mainnet without problems, since it will be backward compatible with non-Dandelion++ nodes, but if you want to see the new feature at work you should probably run a node on the [Monero testnet](https://monerodocs.org/infrastructure/networks/#testnet), where most of the nodes are already running Dandelion++.

Reviewer: ```suggestion
The feature is now part of the code and will be included in the next release of the Monero software, but it's already possible to test it by compiling the CLI software from source (see [instructions on the repository](https://github.com/monero-project/monero#compiling-monero-from-source)). Be aware that to get improved privacy, a node running Dandelion++ needs to communicate with other Dandelion++ capable nodes.
```

I feel this whole paragraph is confusing :/ I removed the last sentences.

> For more detailed information about Dandelion++, how it works and how improves Monero user's privacy, read the article wrote by the Monero Outreach workgroup: ["Dandelion++ for Monero"](https://www.monerooutreach.org/stories/dandelion.html).

Reviewer: ```suggestion
For more detailed information about Dandelion++, how it works and how improves Monero user's privacy, read the article written by the Monero Outreach workgroup: ["Dandelion++ for Monero"](https://www.monerooutreach.org/stories/dandelion.html).
```
